addons/docs/src/frameworks/angular/compodoc.ts

```diff
@@ -169,20 +173,30 @@ export const extractArgTypesFromData = (componentData: Class | Directive | Injec
     data.forEach((item: Method | Property) => {
       const section = mapItemToSection(key, item);
       const defaultValue = isMethod(item) ? undefined : extractDefaultValue(item as Property);
+      const required = isMethod(item) ? false : !item.optional;
       const type =
         isMethod(item) || section !== 'inputs'
           ? { name: 'void' }
           : extractType(item as Property, defaultValue);
+      const decorators = (item.decorators || [])
+        .map(({ name, stringifiedArguments }) => {
+          return `@${name}${stringifiedArguments ? `(${stringifiedArguments})` : ''}`;
+        })
+        .join(' ');
+      const description = [item.description, decorators ? `\`${decorators}\`` : '']
+        .filter(Boolean)
+        .join('<br />');
       const argType = {
         name: item.name,
-        description: item.description,
-        defaultValue,
+        description,
         type,
```

Comment:
I realized that `ArgType.type.required` adds the red *marker in the `ArgsTable` and it makes no sense in all the Angular projects, so this may be addressed in a different way, maybe via JSDoc.